docs/source/user_documentation/credentials_dot_yaml_file.rst

```diff
@@ -0,0 +1,45 @@
+The credentials.yml File
+========================
+
+.. Caution:: Your ``credentials.yml`` file will contain sensitive information. Always store it in a safe place and *never* in source control.
+
+The ``credentials.yml`` file should be configured and moved to a safe place (like a `~/.snowshu` directory).
+
+With your ``credentials.yml`` file relocated you can `configure your replica <replica_dot_yaml_file.html#configure-your-replica>`__.
```

Comment:
It might be worth adjusting the wording here to make note that the credentials file location will be set in `replica.yml` (with the details of that left to the linked page like it currently is)